src/Discovery/Discovery.tsx

```diff
@@ -288,212 +357,208 @@ const Discovery: React.FunctionComponent<DiscoveryBetaProps> = (props: Discovery
-        setModalVisible={setModalVisible}
-        columns={columns}
-        setExportingToWorkspace={setExportingToWorkspace}
-        accessibleFieldName={accessibleFieldName}
-        exportingToWorkspace={exportingToWorkspace}
-        history={props.history}
-      />
-      <Modal
-        className='discovery-modal'
-        visible={modalVisible}
-        onOk={() => setModalVisible(false)}
-        onCancel={() => setModalVisible(false)}
-        width='80vw'
-        footer={false}
-      >
-        <Space style={{ width: '100%' }} direction='vertical' size='large'>
-          { config.studyPageFields.header
-          && (
-            <Space align='baseline'>
-              <h3 className='discovery-modal__header-text'>{modalData[config.studyPageFields.header.field]}</h3>
-              <a href={`/discovery/${modalData[config.minimalFieldMapping.uid]}/`}><LinkOutlined /> Permalink</a>
-            </Space>
-          )}
-          { config.features.authorization.enabled
-          && (modalData[accessibleFieldName]
-            ? (
-              <Alert
-                className='discovery-modal__access-alert'
-                type='success'
-                message={<React.Fragment><UnlockOutlined /> You have access to this study.</React.Fragment>}
-              />
-            )
-            : (
-              <Alert
-                className='discovery-modal__access-alert'
-                type='warning'
-                message={<React.Fragment><LockFilled /> You do not have access to this study.</React.Fragment>}
-              />
-            )
-          )}
-          { config.studyPageFields.fieldsToShow.map((fieldGroup, i) => (
-            <div key={i} className='discovery-modal__attribute-group'>
-              { fieldGroup.includeName
-                  && <h3 className='discovery-modal__attribute-group-name'>{fieldGroup.groupName}</h3>}
-              { fieldGroup.fields.map((field) => {
-              // display nothing if selected study doesn't have this field
-              // and this field isn't configured to show a default value
-                if (!modalData[field.field] && !field.includeIfNotAvailable) {
-                  return null;
-                }
-                return (
-                  <div key={field.name} className='discovery-modal__attribute'>
-                    { field.includeName !== false
-                        && <span className='discovery-modal__attribute-name'>{field.name}:</span>}
-                    <span className='discovery-modal__attribute-value'>
-                      { modalData[field.field]
-                        ? renderFieldContent(modalData[field.field], field.contentType)
-                        : (field.valueIfNotAvailable || 'Not available')}
-                    </span>
-                  </div>
-                );
-              })}
-            </div>
-          ))}
-          { (config.studyPageFields.downloadLinks && config.studyPageFields.downloadLinks.field
```

Comment:
we will need to restore this, since bloodpac is using this feature new in prod (i can help with this)